content/messaging/agent-chat-api/index.mdx

```diff
@@ -1061,8 +1064,8 @@ The list classification is based on threads; 1 chat per 1 thread. Thus, the same
 | ----------------------------------------------------- | -------- | --------- | ----------------------------------------------------------------------------------------- |
 | `filters`                                             | No       | `object`  |                                                                                           |
 | `filters.query`                                       | No       | `string`  |                                                                                           |
-| `filters.date_from`                                   | No       | `string`  | `YYYY-MM-DD` format                                                                       |
-| `filters.date_to`                                     | No       | `string`  | `YYYY-MM-DD` format                                                                       |
+| `filters.from`                                        | No       | `string`  | Date & time format with a resolution of microseconds, UTC string, `YYYY-MM-DDTHH:MM:SS.000000+HH:MM` |
```

Comment:
`YYYY-MM-DDTHH:MM:SS.ssssssZHH:MM` might be better (indicates microseconds and not magic zeroes and also uses `Z` as timezone indicator (instead of a `+`, which can also be `-`) :)